.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+categories:
```

Comment:
The labels in this section come from the labels on pull requests, right?

Reply:
Right. The list is pretty much stable (used in >30 repos).